.github/workflows/publish_website.yaml

```diff
@@ -1,13 +1,18 @@
+name: Build and publish website
```

Comment:
We could have a workflow with two jobs, 1 to create the site and 1 to deploy that artifact. Saves on a bunch of duplication

Reply:
yea that's a good idea, bookmarking this for the next release of the website when we update some of the docs content.